stack-cmbm/recipes/role-vagrant.rb

```diff
@@ -0,0 +1,14 @@
+required_packages = [
+  'libxml2', 'libmysqlclient-dev', 'nodejs', 'autoconf', 'bison', 'build-essential', 'libssl-dev', 'libyaml-dev',
```

Comment:
We have a `nodejs` cookbook that's capable of installing specific versions.

See:
https://github.com/till/easybib-cookbooks/blob/master/stack-scholar/recipes/role-vagrant.rb#L9-L11

Configured via `web-dna.json` in Scholar's Vagrant (Line: 445-451)